File: Python/1281. Subtract the Product and Sum of Digits of an Integer.py
class Solution(object):
    def subtractProductAndSum(self, n):
        """
        :type n: int
        :rtype: int
        """
        sum = 0
        product = 1
        rem = 0
        while n != 0:
            rem = n % 10
            sum = sum + rem
            product = product * rem
            n = n / 10

        return product - sum

File: JavaScript/1281. Subtract the Product and Sum of Digits of an Integer.js
/**
 * @param {number} n
 * @return {number}
 */
var subtractProductAndSum = function (n) {
  let sum = 0;
  let product = 1;

  while (n > 0) {
    let rem = n % 10;
    sum += rem;
    product *= rem;
    n = Math.floor(n / 10);
  }

  return product - sum;
};
